app/jobs/remove_user_event.rb

```diff
@@ -0,0 +1,14 @@
+# Documentation: https://developer.github.com/v3/activity/events/types/#membershipevent
+class RemoveUserJob < ApplicationJob
+  queue_as :github_event
+  def perform(payload)
+    return true unless payload_body.dig('action') == 'removed'
```

Comment:
Would you mind using `payload_body["action"] == "removed"` instead since we're only going one level deep?